### ISSUE TITLE
[Quest] Mount up

### ISSUE THREAD

**Issue by sarahkay88** [WoWManiaUK/Blackwing-Lair#296]
Quest - https://www.wowhead.com/quest=12414/mounting-up
NPC - https://www.wowhead.com/npc=26377
The quest requires you to collect 5 carrots and use has lure on your stick, this worked when getting the first horse, however on the second attempt it let me mount the horse, but when trying to return it, it kept running back to its originally spot. so thought id demount and try a different horse. The problem now is that the horse I demounted follows me were ever I go has if it was one of my pets.
![world of warcraft 23_12_2018 20_46_27](https://user-images.githubusercontent.com/42302580/50387447-736fbb80-06f4-11e9-8395-f73a02f2b1ce.png)

**Comment by sarahkay88** [comment]
![world of warcraft 23_12_2018 20_46_53](https://user-images.githubusercontent.com/42302580/50387453-87b3b880-06f4-11e9-8640-d7fb280473ad.png)

**Comment by Dranah** [comment]
Is the quest itself possible to complete?
Only problem is the horse following you?

**Comment by sarahkay88** [comment]
yes it can be completed if you log in and out after the horde goes on follow. it happened around 3 times.
